src/etc/inc/system.inc

```diff
@@ -73,7 +73,9 @@ function system_sysctl_defaults()
         'debug.kassert.warn_only' => [ 'default' => '1', 'required' => true, 'description' => 'KASSERT triggers a panic (0) or just a warning (1)', 'type' => 'w' ],
         'hw.ibrs_disable' => [ 'default' => '0' ],
         'hw.ixl.enable_head_writeback' => [ 'default' => '0', 'required' => true ],
+        'hw.vtnet.csum_disable' => [ 'default' => '1', 'required' => true ],
         'hw.syscons.kbd_reboot' => [ 'default' => '0' ],
+        'hw.vtnet.csum_disable' => [ 'default' => '1', 'required' => true, 'description' => 'Disable receive and send checksum offload for vtnet(4). Recommended for most KVM based cloud providers.' ],
```

Comment:
remove this (description is auto-parsed from sysctl)

Reply:
Done.